homeassistant/components/matter/strings.json

```
@@ -340,6 +340,15 @@
           "paused": "[%key:common::state::paused%]"
         }
       },
+      "esa_op_out_state": {
+        "name": "Energy optimization opt out",
+        "state": {
+          "no_opt_out": "[%key:common::state::off%]",
+          "local_opt_out": "Local opt-out",
+          "grid_opt_out": "Grid opt-out",
+          "opt_out": "Local and grid opt-out"
```

As "opt-out" is already in the selector name you could drop it from the three options:

```suggestion
          "local_opt_out": "Local",
          "grid_opt_out": "Grid",
          "opt_out": "Local and grid"
```

Makes translations a lot easier, too.